spec/factories/schedule_chains.rb

```diff
@@ -13,10 +13,5 @@
     frequency "Weekly"
     difficulty_rating { rand(3) }
     hilliness { rand(3) }
-
-    after(:create) do |s|
-      d = create(:donation_schedule,schedule_chain:s)
-      r = create(:recipient_schedule,schedule_chain:s)
-    end
```

Comment:
This was causing some tests to fail and doesn't seem to work anymore.  Hard to tell exactly where this went bad - the test suite was failing before this PR.